### ISSUE TITLE
Stepmania Crashes on Boot

### ISSUE THREAD

**Issue by Jopa0226** [stepmania/stepmania#1983]
Latest Version, Windows 10. I literally downloaded the game today. The game goes into the loading/boot screen, then comes up with a crash message.

**Comment by quietly-turning** [comment]
Try following along with this: https://github.com/quietly-turning/Simply-Love-SM5/blob/2d98f1c1f919dc6ecb4bbe1b567e4e40eef372f5/Other/TroubleshootingStepMania-README.md

**Comment by Jopa0226** [comment]
Questions: 1. I have visual studio 2019. Does that matter?
2. I already had the DLL issue, and already downloaded directx9
3. I downloaded it to my external hard drive, not C drive. Does that matter?

**Comment by coderjo** [comment]
What is the crash message? Is there a crashinfo.txt file in the Logs directory?

**Comment by Jopa0226** [comment]
Uhh, yeah. I think there was something like that. Do you want me to send it to you when I'm on my computer?

**Comment by tertu-m** [comment]
Yes. It's difficult to determine what might have caused an error without that file (and sometimes even with it).

**Comment by Jopa0226** [comment]
[crashinfo.txt](https://github.com/stepmania/stepmania/files/4500363/crashinfo.txt)

**Comment by Jopa0226** [comment]
There... there it is.
